src/main/java/gregicadditions/machines/TileEntityProcessingArray.java

```diff
@@ -486,5 +496,35 @@ protected void setActive(boolean active) {
 			}
 		}
 
+		private void computeOutputItemStacks(Collection<ItemStack> recipeOutputs,
```

Comment:
Perhaps rather than copying this from GTCE, we should try to make it public there so we can reuse it?